core/types/legacy_tx.go

```diff
@@ -33,6 +33,11 @@ import (
 	"github.com/ledgerwatch/erigon/rlp"
 )
 
+var (
+	ZeroAddress      = libcommon.HexToAddress("0x0000000000000000000000000000000000000000")
```

Comment:
Isn't this just equivalent to `libcommon.Address{}`?

Reply:
Yes!